examples/evm/simple_transaction.py

```diff
@@ -1,4 +1,4 @@
-from seth import *
+from manticore.seth import *
```

Comment:
we should not encourage `import *`, can you please remove all instances of this?